src/applications/disability-benefits/all-claims/pages/vaMedicalRecords.js

```diff
@@ -18,7 +18,7 @@ export const uiSchema = {
   'view:vaMedicalRecordsIntro': {
     'ui:title': 'VA medical records',
     'ui:description':
-      'Tell us where VA has treated you for your disability. We’ll use the information you provide to help us locate your records and make decisions on your claim.',
+      'Please tell us where VA treated you for your condition.. We’ll use the information you provide to help us locate your records and make decisions on your claim.',
```

Comment:
Did we say we were moving away from using 'please'?

There is also an extra `.` after condition